### PR TITLE
Fixes to web-animations.js for Firefox/Chrome differences.

### PR DESCRIPTION
**Pull request by mithro**

### CODE REVIEW

File: web-animations.js [hunk 1]
```diff
@@ -881,7 +878,39 @@ var TimingEvent = function(token, target, type, localTime, timelineTime, iterati
   this.seeked = seeked ? true : false;
 }
 
-TimingEvent.prototype = Object.create(Event.prototype);
+TimingEvent.prototype = Object.create(Event.prototype, {
+  target: {
+    get: function() {
+      return this._target;
+    },
+    set: function(target) {
```
Comment (dstoc) [inline web-animations.js]: Do we need setters for target/type?

**Comment by dstoc** [comment]
lgtm

File: web-animations.js (hunk 1 above)
Comment (mithro) [inline web-animations.js]: I guess we should just set this._target in the constructor?

Comment (dstoc) [inline web-animations.js]: I think you already did that. Remove these and you can merge.

Comment (mithro) [inline web-animations.js]: Appears I did. Done!

**Pull request merged**